indico_themes_canonical/static/js/default.js

```diff
@@ -171,14 +171,12 @@ function renderCounts(counts, table) {
 /* Renders a table of contents based on the h2 and h3 elements in the page content */
 function renderTableOfContents() {
   const content = document.querySelector(".page-content");
-  if (!content) {
+  const headings = content.querySelectorAll("h2, h3", "h4");
+  // Do not render if there is no content body or less than three headings
+  if (!content && headings.length <= 3) {
```

Comment:
Should this be "or"?

```suggestion
  if (!content || headings.length <= 3) {
```